lib/linguist/languages.yml

```diff
@@ -764,6 +762,16 @@ Clarion:
   - ".clw"
   tm_scope: source.clarion
   language_id: 59
+Classic ASP:
```

Comment:
Wouldn't [`Active Server Pages`](https://en.wikipedia.org/wiki/Active_Server_Pages) be a more appropriate name? I'm only going by a cursory Google search though, so you're probably better-qualified for picking a name than I am.

(For as long as I've worked in web development, I've adamantly refused to touch ASP or anything rhyming with it 😜).

Reply:
Yes, as much as `Hypertext Markup Language` for `HTML` :wink: 

I would say the full name is not common currency, but I'm not sure, I'm not that much into the Microsoft world either :man_shrugging:

Reply:
Fair enough. In any case, if there's a more appropriate name than `Classic ASP`, somebody will eventually let us know. 😂